demo/app/components/autocomplete/autocomplete.component.ts

```diff
@@ -109,17 +113,17 @@ export class AutocompleteComponent implements OnInit {
     this.users$ = this.auto
       .query
       .pipe(
-        startWith(null),
+        startWith(''),
         switchMap((term) => {
           if (term) {
             this.inProgress = true;
             console.warn('searching term: ', term);
             return this.http.get(`https://api.github.com/search/users?q=${term}`)
               .pipe(
                 delay(this.delayApiResponse ? 3000 : 0),
-                map((response: Response) => {
+                map((response: any) => {
```

Comment:
`Response` was throwing a really odd error even though functionally it was working. Since this is just the demo, I punted on it.

Demo _pretty much_ is fully typed.. but this is still one we should revisit at some point.